src/client/eppo-precomputed-client.ts

```diff
@@ -274,15 +291,16 @@ export default class EppoPrecomputedClient {
     );
   }
 
-  private getPrecomputedFlag(flagKey: string): PrecomputedFlag | null {
+  private getPrecomputedFlag(flagKey: string): DecodedPrecomputedFlag | null {
     return this.isObfuscated
       ? this.getObfuscatedFlag(flagKey)
       : this.precomputedFlagStore.get(flagKey);
   }
 
-  private getObfuscatedFlag(flagKey: string): PrecomputedFlag | null {
+  private getObfuscatedFlag(flagKey: string): DecodedPrecomputedFlag | null {
+    const saltedAndHashedFlagKey = getMD5Hash(flagKey, this.decodedFlagKeySalt);
```

Comment:
Something is incorrect about how I'm forming the key to look up the obfuscated flag.

It works fine with the test data added in this PR https://github.com/Eppo-exp/sdk-test-data/pull/89

But it doesn't work when I run a script to use the flags at fs-edge-assignment.eppo.cloud/assignments. There's no match between `saltedAndHashedFlagKey` and any of the fetched obfuscated keys

Would appreciate help here @typotter @rasendubi

Reply:
Actually I just saw this comment https://github.com/Eppo-exp/js-sdk-common/pull/178#discussion_r1883114366

and `appendBinary` seems to fix this issue! I'm going to use that